pages/search.js

```diff
@@ -0,0 +1 @@
+const PAGE_SIZE=2;
```

Comment:
why this?

Reply:
i was working on the other issue as well (forgot to remove the folder :p)

Reply:
oh np!